### ISSUE TITLE
No need for autoconf?

### ISSUE THREAD

**Issue by yurydelendik** [mozilla-spidermonkey/spidermonkey-embedding-examples#58]
I did not install autoconf but was able to build the library. The https://github.com/mozilla-spidermonkey/spidermonkey-embedding-examples/blob/esr91/docs/Building%20SpiderMonkey.md steps do not do anything to use it. Shall we remove the text from the doc?

**Comment by ptomato** [comment]
Yes, I guess since https://github.com/mozilla-spidermonkey/spidermonkey-embedding-examples/pull/56 it is not needed anymore.
